rom/dev/src/rom.ld

```diff
@@ -23,7 +23,9 @@ DCCM_ORG    = 0x50000000;
 MAN1_ORG    = 0x50000000;
 MAN2_ORG    = 0x50001000;
 FHT_ORG     = 0x50002000;
-DATA_ORG    = 0x50003000;
+TBS1_ORG 	= 0x50003000;
```

Comment:
What is the underlying requirement for this change?

Reply:
It's for ease of generation of certificates. Updated the commit message with the explanation.